pkg/image/api/helper.go

```diff
@@ -498,6 +498,10 @@ func ImageWithMetadata(image *Image) error {
 		return fmt.Errorf("unrecognized Docker image manifest schema %d for %q (%s)", manifest.SchemaVersion, image.Name, image.DockerImageReference)
 	}
 
+	// Remove the raw manifest as it's very big and this leads to a large memory consumption in etcd.
```

Comment:
@smarterclayton Fixed. I remove these fields only when you are importing a new image.